### ISSUE TITLE
Multiple rich text Fields - Edit page scroll to bottom on load

### ISSUE THREAD

**Issue by BenBroide** [htmlburger/carbon-fields#381]
When a few rich text fields are set, the edit page will scroll on load to the bottom of page because the cursor "focused" in the last rich text field.
This code on a new installation would reproduce the bug. On edit page the page cursor will focus in the last rich text field which make the page scroll to bottom.

```
Container::make( 'post_meta', 'Custom Data' )
    ->where( 'post_type', '=', 'page' )
    ->add_fields( array(
        Field::make( 'rich_text', 'text1', 'Text 1' ),
        Field::make( 'rich_text', 'text2', 'Text 2' ),
        Field::make( 'rich_text', 'text3', 'Text 3' ),
        Field::make( 'rich_text', 'text4', 'Text 4' ),
        Field::make( 'rich_text', 'text5', 'Text 5' ),
        Field::make( 'rich_text', 'text6', 'Text 6' ),
        Field::make( 'rich_text', 'text7', 'Text 7' ),
        Field::make( 'rich_text', 'text8', 'Text 8' ),
        Field::make( 'rich_text', 'text9', 'Text 9' ),
    ));
```

Same bug happens when switching between tabs, the cursor would focus in the last rich text field and page scrolls to bottom.

**Comment by dmhendricks** [comment]
Semi-related: The focusing has been a (non-mission critical) issue for me as well. I noticed it when I added fields via `term_meta` container - when I visited the page it would always ask if I was "sure that I wanted to navigate away" from the page with any linked that I clicked, even if I didn't make any changes to the field content.

I would just go to the term page and then click any link, and it JS-prompted every time. This only seemed to happen with `rich_text` fields. I noticed this with 2.1.0 (not sure about other releases - it was the version I was using at the time that I noticed it).

The field focus seemed to be telling the browser that data was changed even when that was not the case. I resolved it by moving the fields to a different container (which I ultimately decided made more sense for the user experience anyway, so no worries), however, it was quite disruptive at the time.
